packages/bus-sqs/src/sqs-transport.ts

```diff
@@ -378,7 +385,11 @@ export class SqsTransport implements Transport<SQSMessage> {
     })
 
     try {
-      await this.sqs.send(command)
+      if (this.autoProvision) {
+        await this.sqs.send(command)
```

Comment:
could the command declaration be scoped inside autoProvision too?

eg:

```suggestion
        const command = new CreateQueueCommand({
          QueueName: queueName,
          Attributes: queueAttributes
        })
        await this.sqs.send(command)
```